src/main/webapp/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html> 
+<html>
+    <head>
+        <title>CRAM</title>
+        <script src="https://apis.google.com/js/platform.js?onload=init" async defer></script>
+        <script src="https://apis.google.com/js/client:plusone.js" type="application/javascript"></script>
+        <script src="js/config.js"></script>
+        <script src="js/script.js"></script>
+    </head>
+    <body>
+        <h1>CRAM</h1>
+        <button onclick='authenticate().then(loadClient)'>authenticate</button>
+        <input id="ytVideoSearch" type="text" />
+        <!--ovJcsL7vyrk-->
```

Comment:
What does this comment signify?

Reply:
This was just a track id that I was using for testing.